wgsl/index.bs

```diff
@@ -11604,6 +11678,18 @@ Here is the list of exceptions:
         - [=ReturnValueMayBeNonUniform=] if the argument corresponding to the `t` parameter
             is a [=type/read-write storage texture=]
         - [=NoRestriction=] otherwise
+- A call to a function in
+    [[#subgroup-builtin-functions]] or [[#quad-builtin-functions=]]:
+    - Has a [=function tag=] of [=ReturnValueMayBeNonUniform=].
+    - Has a [=call site tag=] as follows:
+        - Let *DF* be the [=nearest enclosing diagnostic filter=] for the call site location and triggering rule [=trigger/subgroup_uniformity=]
+        - If *DF* exists, then let *S* be the *DF*'s new severity parameter.
+            - If *S* is the severity [=severity/off=], the call site tag is [=CallSiteNoRestriction=].
+            - Otherwise, the call site tag is [=CallSiteRequiredToBeUniform.S=], with [=potential-trigger-set=]
+                consisting of a [=trigger/subgroup_uniformity=] element.
+        - If there is no such *DF*,
+            the call site tag is [=CallSiteRequiredToBeUniform.S|CallSiteRequiredToBeUniform.error=], with [=potential-trigger-set=]
+            consisting of a [=trigger/subgroup_uniformity=] element.
```

Comment:
Also need to mention the `ParameterRequiredToBeUniform` tag for the `delta` parameters on `subgroupShuffle*` builtins.